.circleci/config.yml

```diff
@@ -28,15 +28,15 @@ jobs:
             - run:
                 name: Set Max Memory
                 command: |
-                    NODE_OPTIONS=--max_old_space_size=6144
+                    NODE_OPTIONS=--max_old_space_size=5120
```

Comment:
I'm still not sure what this max memory stuff is doing. What is this change doing? And why do we need it?

Reply:
Makes sense... I'm happy to merge this since it's working, but it might be worthwhile to revisit this stuff across all repos once we have a license for circleci.